templates/astro-init/src/pages/index.astro

```diff
@@ -1,16 +1,77 @@
 ---
-
+import Layout from '../layouts/Layout.astro';
+import '../assets/main.css';
 ---
 
-<html lang="en">
-	<head>
-		<meta charset="utf-8" />
-		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
-		<meta name="viewport" content="width=device-width" />
-		<meta name="generator" content={Astro.generator} />
-		<title>Astro</title>
-	</head>
-	<body>
-		<h1>Astro</h1>
-	</body>
-</html>
+<Layout title="Welcome to Astro.">
+	<section>
+		<div class="header">
+			<img src='azion.png' width="100" height="24" />
+		</div>
+		<div class="content">
+			<h1 class="text-gradient">Astro</h1>
+			<h1>Welcome to Astro</h1>
+			<p class="instructions">
+				To get started, open the directory <code>src/pages</code> in your project.<br /><br />
+				<strong>Code Challenge:</strong> Tweak the "Welcome to Astro" message above.
+			</p>
+		</div>
+		<div class="footer">
+			<a href="https://www.azion.com/en/documentation/" target="_blank">
+			<h1>Docs</h1>
+			<p>
+				Besides providing structure, it allows interactions to occur with the surface of the element, thus enabling it to have states.
+			</p>
+			</a>
+			<a href="https://medium.com/aziontech" target="_blank">
+			<div>
+				<h1>Medium</h1>
+				<p>
+				Dive deep into our platform's use cases on Medium and join a community where developers connect, collaborate, and innovate.
```

Comment:
@MarianaAguilera does it make sense to take people to medium considering our goal is to take users to our blog? Another thing is that the use cases will be in our blog + docs.

Reply:
Pessoal essa definição de interface foi feita no passado com intuito de crescermos a visualização do Discord/Twitter/Medium era um objetivo de DevX, para mudarmos precisamos aprovar layout novo e aplicar reversamente para todos os templates, não vejo isso, neste momento como blocker.